components/book/List.tsx

```diff
@@ -29,7 +29,44 @@ export const List: FunctionComponent<Props> = ({ books }) => (
 			<tbody>
 				{books &&
 					books.length !== 0 &&
-					books.map((book) => <ListItem key={book["@id"]} book={book} />)}
+					books.map((book) => (
+						<tr key={book["@id"]}>
+							<th scope='row'>
+								<ReferenceLinks items={book["@id"]} type='book' />
+							</th>
+							<td>{book["isbn"]}</td>
```

Comment:
Maybe could we loop over all properties instead? I'm not 100% sure if it's a good idea or not (less code which is good, but maybe harder to customize).

Reply:
In fact its a loop in the generator.